tests/test_parametric_components/test_PortCutterCircular.py

```diff
@@ -1,19 +1,56 @@
-
+import math
 import unittest
 
 import paramak
 
+import pytest
+
+
+class test_component(unittest.TestCase):
```

Comment:
Code inspector wants it to be PascalCase style
```suggestion
class TestComponent(unittest.TestCase):
```